dynamic_rest/filters.py

```diff
@@ -283,7 +285,9 @@ def _filters_to_query(self, includes, excludes, serializer, q=None):
         def rewrite_filters(filters, serializer):
             out = {}
             for k, node in six.iteritems(filters):
-                filter_key = node.generate_query_key(serializer)
+                filter_key, field = node.generate_query_key(serializer)
+                if isinstance(field, (BooleanField, NullBooleanField)):
+                    node.value = node.value.lower() not in self.FALSEY_STRINGS
```

Comment:
Should we try to validate/prevent non-boolean-looking values here?

E.g. if I make q